### ISSUE TITLE
Main CLI runtime not showing in documentation

### ISSUE THREAD

**Issue by reisroni** [citrusframework/citrus#1242]
**Citrus Version**
3.4.0 - 4.3.3

**Expected behavior**
Instructions how to run citrus main in the documentation.

**Actual behavior**
Instructions are not there (both HTML and pdf), the box is empty.

**Test case sample**
[Main CLI runtime](https://citrusframework.org/citrus/reference/4.1.1/html/index.html#runtime-main)

![Citrus Main CLI](https://github.com/user-attachments/assets/6f1c28dd-31fa-4460-9fbf-5ddab92ffbe1)

**Comment by bbortt** [comment]
hasn't been there for a while (source linked).. 😉 thoughts @christophd?

[![2024-10-27-184441_3840x2160_scrot](https://github.com/user-attachments/assets/4f42da6b-85ea-437f-9209-cad45ddbe05c)
](https://github.com/citrusframework/citrus/blame/main/src/manual/runtimes-main.adoc)

**Comment by christophd** [comment]
yes, docs for Main CLI are still under construction and incomplete. This is mainly due to Citrus JBang coming in which is the much more powerful way of running Citrus tests from the command line.

Please consider having a look into Citrus JBang: https://citrusframework.org/citrus/reference/html/index.html#runtime-jbang

**Comment by christophd** [comment]
Should be fixed with latest 4.5.0 documentation. Also consider to have a look into the new Citrus JBang support
